common/dtd/ldml.dtd

```diff
@@ -2126,6 +2126,15 @@ $Revision$
 <!ATTLIST minusSign numberSystem CDATA #IMPLIED >
     <!--@DEPRECATED-->
 
+<!ELEMENT approximatelySign ( #PCDATA ) >
```

Comment:
You also need to add this to the parent element, "symbols", in the right order.

Reply:
Done